packages/service-integration/src/service_integration/osparc_image_specs.py

```diff
@@ -2,7 +2,9 @@
 
 """
 
+from typing import Any
 
+from service_integration._compose_spec_model_autogenerated import ListOrDict
```

Comment:
we never import directly from the auto-generated but rather the intermediate `servcie_integration.compse_spec_model`. This way we keep control of what is exposed or not